dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx

```diff
@@ -98,6 +105,12 @@ function decidePosition(role: string, design: ArticleDesign) {
 	return role === 'supporting' ? partiallyLeft : partiallyInline;
 }
 
+const decideFontWeight = (theme: ArticleTheme) => {
+	if (theme === ArticleSpecial.SpecialReportAlt) return 'light';
+
+	return 'bold';
+};
```

Comment:
Take it or leave it, but:

```suggestion
const decideFontWeight = (theme: ArticleTheme) => (theme === ArticleSpecial.SpecialReportAlt) ? 'light' : 'bold';
```